packages/esm-reports-app/src/components/edit-scheduled-report/edit-scheduled-report-form.component.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { first } from 'rxjs/operators';
+import styles from './edit-scheduled-report-form.scss';
+import SimpleCronEditor from '../simple-cron-editor/simple-cron-editor.component';
+import { useReportDefinition, useReportDesigns, useReportRequest, runReportObservable } from '../reports.resource';
+import ReportParameterInput from '../report-parameter-input.component';
+import { Button, ButtonSet, Form, Select, SelectItem, Stack } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { showToast, useLayoutType } from '@openmrs/esm-framework';
+
+interface EditScheduledReportForm {
+  reportDefinitionUuid: string;
+  reportRequestUuid: string;
+  closePanel: () => void;
+}
+
+const EditScheduledReportForm: React.FC<EditScheduledReportForm> = ({
+  reportDefinitionUuid,
+  reportRequestUuid,
+  closePanel,
+}) => {
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+
+  const reportDefinition = useReportDefinition(reportDefinitionUuid);
+  const { reportDesigns } = useReportDesigns(reportDefinitionUuid);
+  const { reportRequest } = useReportRequest(reportRequestUuid);
+
+  const [reportParameters, setReportParameters] = useState({});
+  const [renderModeUuid, setRenderModeUuid] = useState<string>();
+  const [initialCron, setInitialCron] = useState<string>();
+  const [schedule, setSchedule] = useState<string>();
```

Comment:
```suggestion
  const [renderModeUuid, setRenderModeUuid] = useState();
  const [initialCron, setInitialCron] = useState();
  const [schedule, setSchedule] = useState();
```

TypeScript will automatically infer these as `string | undefined` from usage.

Reply:
fixed